data/presets/highway/path/crossing/_traffic_signals.json

```
@@ -0,0 +1,30 @@
+{
+    "icon": "temaki-railway_signals",
+    "fields": [
+        "crossing",
+        "{@templates/crossing/traffic_signal}",
+        "{@templates/crossing/markings}",
+        "{@templates/crossing/defaults}",
+        "surface"
+    ],
+    "moreFields": [
+        "{@templates/crossing/traffic_signal_more}",
+        "{@templates/crossing/geometry_way_more}",
+        "{@templates/crossing/bicycle_more}",
+        "flashing_lights"
+    ],
+    "geometry": [
+        "line"
+    ],
+    "tags": {
+        "highway": "path",
+        "path": "crossing",
+        "crossing": "traffic_signals"
+    },
+    "reference": {
+        "key": "crossing",
+        "value": "traffic_signals"
+    },
+    "name": "Crossing Cycle & Foot With Pedestrian Signals",
```

```suggestion
    "name": "Cycle & Foot Crossing With Pedestrian Signals",
```
I think this would sound a bit more natural

---

Thanks!